src/hooks/__tests__/useTektonResults.spec.ts

```diff
@@ -25,12 +38,24 @@ const mockResponseNext = [
   {},
 ] as [any[], RecordsList];
 
+const mockTaskRun = {
+  apiVersion: 'v1Alpha1',
+  spec: {},
+  kind: 'TaskRun',
+  metadata: { uid: 'test-id' },
+};
+
+const mockResponseLogs = 'test-log';
+
 const getPipelineRunsMock = getPipelineRuns as jest.Mock;
 const getTaskRunsMock = getTaskRuns as jest.Mock;
-const getTaskRunLogMock = getTaskRunLog as jest.Mock;
+// const getTaskRunLogMock = getTaskRunLog as jest.Mock;
```

Comment:
Was this meant to be left commented out?

Reply:
ya, will remove this